phing/tasks/setup.xml

```diff
@@ -248,13 +248,17 @@
           <param>drush</param>
         </drush>
         <!-- Revert all features. -->
-        <drush command="fra" assume="yes" alias="${drush.alias}">
-          <option name="bundle">${cm.features.bundle}</option>
-        </drush>
+        <foreach list="${cm.features.bundle}" param="bundle" target="setup:update:features-import" />
```

Comment:
Testing confirmed the following forms all feed validly to the ForEachTask:

single:
```yaml
cm:
  features:
    bundle: my_feature_bundle
```
comma-delimited:
```yaml
cm:
  features:
    bundle: my_feature_bundle, my_other_feature_bundle, my_last_feature_bundle
```
per-line:
```yaml
cm:
  features:
    bundle: 
      - my_feature_bundle
      - my_other_feature_bundle
      - my_last_feature_bundle
```
My personal preferences is for the last option, but it looks like Phing is happy to take either form.